suse_migration_services/prechecks/kernels.py

```diff
@@ -0,0 +1,69 @@
+# Copyright (c) 2022
+#
+# SUSE Linux LLC.  All rights reserved.
+#
+# This file is part of suse-migration-services.
+#
+# suse-migration-services is free software: you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# suse-migration-services is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with suse-migration-services. If not, see <http://www.gnu.org/licenses/>
+#
+"""Pre-Checks multiversion kernels
+
+Check to see if multiversion.kernels are enabled in /etc/zypp/zypp.conf and
+if there is more than one kernel currently installed. If detected a warning is reported
+as multiversion.kernels needs to be disabled and old kernels cleaned up before
+migrationg."""
+
+# import os
```

Comment:
may want to remove this commented out line

Reply:
thanks will remove